packages/react/package.json

```diff
@@ -28,6 +28,7 @@
     "start": "react-scripts start",
     "storybook": "start-storybook -p 9001 -c .storybook",
     "storybook-build": "build-storybook -c .storybook -o storybook-build",
+    "storybook-deploy": "build-storybook -c .storybook -o storybook-build  && cd storybook-build && surge build",
```

Comment:
Nitpicking: the last two commands can collapse into `surge build storybook-build`